src/modules/pece/pece_groups/pece_groups.module

```diff
@@ -148,25 +147,12 @@ function _pece_groups_edit_form_fields_invisible(&$form, $fields = array()) {
   }
 }
 
-/**
- * Hide the field_pece_project_ref (deprecated) to keep its records for future use if needed.
- * @param $form
- * @param array $fields
- */
-function _pece_groups_hide_project_ref_field(&$form, $fields = array()) {
```

Comment:
@revagomes
I had to remove this code for the existing project field to work.

Check carefully if you could do this

Reply:
That's totally fine. It was used to hide the field on node forms and skip any validation once it wasn't in use at that time.